Mathlib/RingTheory/IntegralClosure/Algebra/Basic.lean

```diff
@@ -228,3 +229,30 @@ instance Algebra.IsIntegral.prod [Algebra.IsIntegral R A] [Algebra.IsIntegral R
     (Algebra.isIntegral_def.mp ‹_› x.1).pair (Algebra.isIntegral_def.mp ‹_› x.2)
 
 end
+
+section BaseChange
+
+variable {R A : Type*} [CommRing R] [CommRing A] [Algebra R A]
+variable (B : Type*) [CommRing B] [Algebra R B]
+
+open scoped TensorProduct in
+lemma Algebra.IsIntegral.base_change [Algebra.IsIntegral R A] :
+    Algebra.IsIntegral B (B ⊗[R] A) := by
```

Comment:
This is [Algebra.IsIntegral.tensorProduct](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/IntegralClosure/IsIntegralClosure/Basic.html#Algebra.IsIntegral.tensorProduct)

```lean
instance Algebra.IsIntegral.tensorProduct
    (R : Type u_1) (A : Type u_2) (S : Type u_4) [CommRing R] [CommRing A] [CommRing S]
    [Algebra R A] [Algebra R S] [int : Algebra.IsIntegral R S] :
  Algebra.IsIntegral A (TensorProduct R A S)
````

Reply:
In general you can use [Loogle](https://loogle.lean-lang.org/?q=open+TensorProduct+in+Algebra.IsIntegral+%3FB+%28%3FB+⊗%5B%3FR%5D+%3FA%29) to discover such theorems.